fsi-workflow/01_solidMesh/tasks.md

```diff
@@ -1,182 +1,147 @@
+- Click `Mesh` -> `FEM Mesh group`:
 
- - repeat for other surfaces (they are **4**: upper, lower, tip and trailing. pay attention to trailing edge surface)
- - rename the group to **wetSurface**
+![FreeCAD: Mesh group](./images/Groups02.png)
 
- - repeat for tip surface (it will be used for single physics test)
- - rename the group to **tip**
+  In case you cannot select anything and you get an error "Active Task Dialog found!", you might need to switch to the `Tasks` tab and click `OK` or `Cancel` to exit from the previous task.
 
-Select the mesh->double click-> press apply to remesh and create groups-> ok
+- In the `Model` tab, there should now also be a `MeshGroup` under the `FEMMeshGmsh`.
 
-==IMPORTANT==: you need to remesh otherwise the groups won't be created
+![FreeCAD: Added MeshGroup](./images/Groups03.png)
 
-## Save INP file and verify
+- In the `Tasks` tab, select `Label` as `Identifier used for mesh export` and `Face, Edge, Vertex` as `Selection mode`:
 
- - keep the mesh selected
- - File->export 
- - name the file  **wing2312.inp**
- - save
- - close
- - File->Save the FreeCAD model
+![FreeCAD: Add surface to mesh group](./images/Groups04.png)
 
-open the **inp** file with a text editor:
+- Click `Add`, then click on the rendering to select the profile of the wing (pay attention to reference frame to identify it), and `OK` to add the surface to the mesh group. You can rotate the view using the cude on the upper right corner, or [using your mouse](https://wiki.freecad.org/Mouse_navigation) (e.g., by `Shift` + `right click`).
+- In the `Model` tab, select the `MeshGroup` and rename its `Label` to `root`.
 
+![FreeCAD: Rename mesh group](./images/Groups05.png)
 
- - look for **ELSET**, skip Evolume, you may remove remove all other ELSET
- - look for **NSET**, skip Nall
-     - *NSET, NSET=tip_Nodes 
-     - *NSET, NSET=wetSurface_Nodes
-     - *NSET, NSET=root_Nodes
+- Repeat for other surfaces (they are 4: `upper`, `lower`, `tip` and `trailing`. Pay attention to the trailing edge surface, you need to zoom-in to see it). Remember to first select `FEMMeshGmsh`.
+- Rename the group (**TODO: Which? How?**) to `wetSurface`
+
+- Repeat for the leading edge (it will be used for single physics tests), with group name `tip`.
+
+**TODO:** Add screenshot with final setup
+
+Select the mesh -> double click -> click `Apply` to remesh and create groups -> click `OK`.
+
+**IMPORTANT**: You need to remesh otherwise the groups won't be created.
+
+## Export the mesh file and verify
+
+- Keep the mesh selected.
+- Click `File` -> `Export...`.
+- Name the file  `wing2312.inp`.
+- Select `FEM mesh formats (*.dat, *.inp, ...)`.
+- Click `Save`.
+
+Save also the FreeCAD model with `File` -> `Save`.
+
+To verify, open the `wing2312.inp` file you just generated with a text editor:
+
+- Look for `ELSET`: Skip the `Evolume` set, you may remove all other `ELSET` entries (but you don't need to, they are just not used in the rest of the course)
```

Comment:
Do we actually need to delete these `ELSET` entries, or is it fine to keep them there?
I would like to avoid manual editing of the mesh in a text editor. Sounds hacky.

Reply:
I am trying to understand the best way to proceed. It should be safe to keep them. I agree that it is not nice to manually edit the mesh file.

Reply:
Sure, it is a sort of WIP there. I was still looking for possible issues in further steps. For example, Aster defaults the same name for a face group and the group of nodes for the same face, but it is able to resolve. This appears not to be the case here, and I'll update the task.